dist/deb/debian/rules.in

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/make -f
-BUILD_ROOT := $(CURDIR)/debian/rats-tls
+BUILD_ROOT := $(CURDIR)/debian/rats-tls-host
```

Comment:
why append "host"  as suffix?

Reply:
We no longer build the package of rats-tls, but build the package of rats-tls in host、tdx、occlum and sgx mode, so it is named here as rats-tls-host